### ISSUE TITLE
supersid_scanner.py in combination with textSidViewer throws exception

### ISSUE THREAD

**Issue by fenrog** [sberl/supersid#22]
supersid.cfg:
```
[PARAMETERS]
viewer = text
```
start as:
`python3 supersid_scanner.py`

Exception:
```
Exception in thread Thread-10:
Traceback (most recent call last):
File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
File "/usr/lib/python2.7/threading.py", line 1073, in run
    self.function(*self.args, **self.kwargs)
File "/home/pi/supersid/supersid/sidtimer.py", line 70, in _ontimer
    self.callback()
File "./supersid_scanner.py", line 115, in on_timer
    self.viewer.clear()
AttributeError: textSidViewer instance has no attribute 'clear'
```

**Comment by fenrog** [comment]
supersid_scanner.py doesn't read the configuration file to decide on the viewer. It always uses  textsidviewer.
These two lines have to be deleted. The its sucesfully saves scanner_buffers.raw.ext.yyyy-mm-dd.csv

```
        # clear the View to prepare for new data display
        self.viewer.clear()
```

**Comment by fenrog** [comment]
fixed with 52a0c9fbb9f9cc3a428999730b4bfeefbd5fad79
